test/01_cond_field/process_simple.py

```diff
@@ -64,11 +64,10 @@ def __init__(self):
 
     def process(self):
         sample_storage = SampleStorageHDF(file_path=os.path.join(self.work_dir, "mlmc_{}.hdf5".format(self.n_levels)))
-        sample_storage.chunk_size = 1024
+        sample_storage.chunk_size = 1e8
```

Comment:
It is not effective to have a small chunk in this place, it takes a long time.

Reply:
Could take longer when usin very small chunks, but size about 1e5 should make no difference. 
Nevertheless the point was, that this may serve as a real test of the chunk-based estimate algorithm.